lib/libutee/include/tee_api_defines.h

```diff
@@ -70,6 +70,7 @@
 #define TEE_ORIGIN_COMMS                0x00000002
 #define TEE_ORIGIN_TEE                  0x00000003
 #define TEE_ORIGIN_TRUSTED_APP          0x00000004
+#define TEE_ORIGIN_CLIENT_APP           0x00010000
```

Comment:
This falls in the GlobalPlatform reserved range (TEE Internal Core API v1.2.1 section 4.2.3). Please use `0xF0000001` instead.
Also `lib/libutee/include/tee_api_defines_extensions.h` would be a better place.

Reply:
Thank you for catching that! Changed and moved.